source/server/server.h

```diff
@@ -54,6 +54,7 @@ namespace Server {
  */
 #define ALL_SERVER_STATS(COUNTER, GAUGE, HISTOGRAM)                                                \
   COUNTER(debug_assertion_failures)                                                                \
+  COUNTER(envoy_bug_failures)                                                                      \
```

Comment:
I hear we document these counters somewhere in the docs.

Reply:
:) done, along with release notes